### ISSUE TITLE
JSONP support broken in Scalatra 2.3.0 with json4s 3.2.11

### ISSUE THREAD

**Issue by maximilianschweitzer** [scalatra/scalatra#466]
In Scalatra 2.3.0 with json4s-jackson 3.2.11 JSONP support is broken. After downgrading to json4s-jackson 3.2.10 everything works again.

After setting
```
override val jsonpCallbackParameterNames: Iterable[String] = Some("callback")
```
and calling the url with ?callback=xyz you receive a NoSuchMethodError.

The problem is the line 
```
writer.write("%s(%s);".format(some, compact(render(transformResponseBody(jv)))) 
```
within org.scalatra.JsonOutput.

The render method is not found.

**Comment by xuwei-k** [comment]
Scalatra 2.3.0 depends on json4s 3.2.10
https://github.com/scalatra/scalatra/blob/v.2.3.0/project/build.scala#L360
Json4s 3.2.10 and 3.2.11 are not binary compatible.

**Comment by maximilianschweitzer** [comment]
Okay, thx. Now I know why I did use the incorrect version. 
Scalatra 2.4 is not released yet, but if you click on the documentation link on the website you are redirected to the Scalatra 2.4 documentation. In this documentation it says Json4s 3.2.11 and scalatra 2.4. Because I got a dependency error (cannot resolve scalatra 2.4) I thought of a typo and corrected it to 2.3.
Perhaps it would be better if the documentation points to the current version or at least is showing a scalatra version inside the dependencies list that really exists.

**Comment by dozed** [comment]
Good point. You can also send a pull request to 
https://github.com/scalatra/scalatra-website if you find an error in the 
documentation.

Am 08.02.2015 um 14:58 schrieb Maximilian Schweitzer:

**Comment by seratch** [comment]
The upcoming 2.3.1 release will fix this issue because 2.3.1 will be built with json4s 3.2.11.

**Comment by seratch** [comment]
scalatra 2.3.1 is out.
